_episodes/39.md

```diff
@@ -0,0 +1,273 @@
+
+## Releases 392 to 393
+
+Official highlights from Martin Traverso:
+
+[Trino 392](https://trino.io/docs/current/release/release-392.html)
+
+* Support for dynamic filtering with fault-tolerant query execution.
+* Support for correlated subqueries in `DELETE` queries.
+* Support for Amazon S3 Select pushdown for JSON files.
+* Support for Avro format in Iceberg connector.
+* Faster queries when filtering by `__time` column in Druid.
+
+[Trino 393](https://trino.io/docs/current/release/release-393.html)
+
+* Add support for `MERGE`.
+* Improved performance of highly selective `LIMIT` queries.
+* Experimental docker image for `ppc64le`.
+* Dynamic filtering support for various connectors.
+* Support for JSON and bytes type in Pinot.
+
+Additional highlights worth a mention according to Manfred:
+
+* Lots of other improvements on Delta Lake, Hive, and Iceberg connectors.
+* Merge support in a bunch of connectors.
+* OAuth 2.0 refresh token fixes
+
+More detailed information is available in the release notes for
+[Trino 392](https://trino.io/docs/current/release/release-392.html),
+and
+[Trino 393](https://trino.io/docs/current/release/release-393.html).
+
+## Concept of the episode: Trino at Raft
+
+Raft provides consulting services and is particularly skilled at DevSecOps. One
+particular challenge they face is dealing with fragmented government
+infrastructure. In this episode, we dive in to learn how Trino enables Raft to
+supply government sector clients with a data fabric solution. Raft takes a
+special stance on using and contributing to open source solutions that run well
+on the cloud.
+
+### Intro Software Factories
+
+A “software factory” is an organized approach to software development that
+provides software design and development teams a repeatable, well-defined path
+to create and update software. It results in a robust, compliant, and more
+resilient process for delivering applications to production.
+([Source](https://tanzu.vmware.com/software-factory))
+
+This is a push against the previous attempts from larger government contractors
+who tried to build one-size-fits-all solutions that ultimately failed. The new
+wave of government solutions relies on methodologies similar to the software
+industry that append more rules and standards around technologies they can adopt
+in the stack.
+
+This is now a common practice for government agencies to use, as they are able
+to take standardized software stacks that go through rigorous validation to make
+sure the meet the standards of the government. One important element to these
+stacks are that they can be deployed in virtually any environment. A common
+way to do this is using Kubernetes and containers.
+
+### Standards and anatomy of a stack
+
+With the movement towards standardization, government contractors will generally
+build their stack using Kubernetes templates. Kubernetes is the underpins of
+each of these stacks while monitoring, telemetry, and policy agents are layered
```

Comment:
```suggestion
each of these stacks while telemetry, monitoring, and policy agents are layered
```
This ordering makes it more clear that 'monitoring' is a gerund when first reading it. With the original ordering, it may be read as a verb by some readers and feel a little jarring.

Reply:
wow...that's pretty neat...I like it :)